portfolio/src/main/java/com/google/sps/data/BookAPI.java

```diff
@@ -0,0 +1,36 @@
+//  BookAPI.java
+//  Sample Google API
+//
+//  Created by Mohammadreza on 10/7/18.
+//  Copyright © 2019 Mohammadreza Mohades. All rights reserved.
+
+package com.google.sps.data;
+
+import com.google.sps.data.FullBook;
+import com.google.sps.data.RequestJson;
+import java.io.UnsupportedEncodingException;
+import java.net.URLEncoder;
+import java.util.ArrayList;
+
+public class BookAPI {
+  public ArrayList<FullBook> search(String query, int numberResults) {
+    if (query.equals(""))
+      return null;
```

Comment:
nit: Style guide asks for braces :)
https://google.github.io/styleguide/javaguide.html#s4.1.1-braces-always-used